WordPress/gutenberg
# Global Styles: Adds site background color controls component in global styles

## What?
Builds upon https://github.com/WordPress/gutenberg/pull/63216

## Why?
As mentioned in https://github.com/WordPress/gutenberg/pull/63216#issuecomment-2330295914 and to organise the component structure of the background-color control in global styles.

## How?
- Adds a background-color component for site global styles
- Adds upload icon for background-image and changes it's shape to rounded-rectangle 

## Testing Instructions
1. Open the site editor by going Appearance > Editor
2. Go to Styles and see the Background option
3. In the Background there should be two controls, Image and Color

## Screenshots or screencast 

https://github.com/user-attachments/assets/1071690d-a7a5-46b9-af33-f71f3119a0f5
